projects/e2fsprogs/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
+case $SANITIZER in
```

Comment:
is this part really necessary? could you please check the configure script and see what flags do these options enable?

in general, we prefer project not to use any sanitizers options themselves -- just use provided `CFLAGS` and `CXXFLAGS`, otherwise the build integration could be fragile

Reply:
I had initially tried to build it without this, but kept getting errors related to undefined addrsan functions (when I built with addrsan enabled). My host and I decided that this was the way to go in order to get around these errors. Please let me know if you have a better solution.

Reply:
without seeing the actual errors, I suspect that the issue is missing `LDFLAGS` as per this line https://github.com/tytso/e2fsprogs/blob/8fd92e9af006da392d667775b8bbe0db9e8639d2/configure#L14118

can you please try something like `export LDFLAGS="$CXXFLAGS"` before the configure step?

Reply:
It works, thanks!